### PR TITLE
[VEN-1303] add simulations for provide VAI/USDT liquidity VIP in PancakeSwap

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description
VIP-106 Provide Liquidity in Pancake Swap

>         withdraw 180K VAI to the Normal Timelock contract from the treasury
>         withdraw 180K USDT to the Normal Timelock contract from the treasury
>         approve 180K of the VAI balance in the contract to the PCS router
>         approve 180K of the USDT balance in the contract to the PCS router
>         invoke addLiquidity function on the PCS router, with the following parameters:
>         tokenA: (VAI) 0x4bd17003473389a42daf6a0a729f6fdb328bbbd7
>         tokenB: (USDT) 0x55d398326f99059ff775485246999027b3197955
>         amountADesired: 180K
>         amountBDesired: 180K
>         amountAMin: 178.2K VAI (1 USDT = 0.99 VAI)
>         amountBMin: 178.2K USDT (1 USDT = 1.01 VAI)
>         to: (VTreasury) 0xf322942f644a996a617bd29c16bd7d231d9f35e9

Resolves VEN

### CODE REVIEW

File: simulations/vip-106/simulations.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  const provider = ethers.provider;
+  const currentLiquidityUsdt = parseUnits("90.294422022310547114", 18);
+  const currentLiquidityVai = parseUnits("87.950866165080484144", 18);
+
+  before(async () => {
+    pair = new ethers.Contract(PAIR, PAIR_ABI, provider);
+  });
+
+  testVip("VIP-106 Provide Liquidity in Pancake Swap", vip106());
+  describe("Post-VIP behavior", async () => {
```
Comment (chechu) [inline simulations/vip-106/simulations.ts:22]: We can add another test to check the balance of the LP in the treasury, verifying it is increased after providing the liquidity

Comment (GitGuru7) [reply simulations/vip-106/simulations.ts:25]: Added [here](https://github.com/VenusProtocol/vips/pull/5/commits/248b385fc31cba942290dc719daf9149b17d2d4d)

**Review by chechu — APPROVED** [review]
LGTM

**Review by defcon022 — APPROVED** [review]

**Comment by chechu** [comment]
VIP launched: https://app.venus.io/governance/proposal/108

**Pull request merged**